src/envo/envo-edit.owl

```diff
@@ -41163,6 +41165,23 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#inSubset> <htt
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105126> "weathering of plastic"@en)
 SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105126> <http://purl.obolibrary.org/obo/ENVO_06105119>)
 
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105201> (water table)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/w/index.php?title=Water_table&oldid=1049566132") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105201> "The water table is the surface where the water pressure head is equal to the atmospheric pressure."@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105201> <https://orcid.org/0000-0001-6284-4821>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105201> "2021-10-29T14:15:55Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105201> "water table"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105201> <http://purl.obolibrary.org/obo/ENVO_01001784>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_06105203> (water table depth)
+
+AnnotationAssertion(<http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_06105203> "A depth of the water table"@en)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_06105203> <https://orcid.org/0000-0001-6284-4821>)
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/date> <http://purl.obolibrary.org/obo/ENVO_06105203> "2021-11-04T17:43:01Z"^^xsd:dateTime)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_06105203> "water table depth"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105203> <http://purl.obolibrary.org/obo/PATO_0001595>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105203> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000055> <http://purl.obolibrary.org/obo/ENVO_06105201>))
```

Comment:
BFO:realizes doesn't make sense here. RO's `characteristic of` (formerly `inheres in`) or its subclass `quality of` would fit.

```suggestion
SubClassOf(<http://purl.obolibrary.org/obo/ENVO_06105203> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0000080> <http://purl.obolibrary.org/obo/ENVO_06105201>))
```